lib/fathead/node_js/redirector.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+output_txt='./output.txt';
+redirect_txt='./redirects.txt';
+
+#get all the titles from the output.txt
+awk '{print $1}' ${output_txt} > temp_names
+
+rm -f ${redirect_txt}
+
+#create redirect for . seperation
+cat temp_names |  while read -r ;
+do
+	temp_redirect=$(echo ${REPLY} | sed -e 's/\./ /g');
+	[ "$temp_redirect" != "$REPLY" ] && echo "$temp_redirect, $REPLY" >> $redirect_txt;
+done;
+
+#create redirect for camel case
```

Comment:
Fathead matching is case insensitive unless multiple articles with identical titles, but different casing exist. I don't think we need to worry about that here.